### PR TITLE
 4 0.8.8 Preflight Core

### PR DESCRIPTION
**Pull request by henry-richard**
4 0.8.8 Preflight Core

### CODE REVIEW

**Comment by henry-richard** [comment]
Hey **Wanda Carlson**, can you check this?

**Comment by henry-richard** [comment]
Do you want me to mge it?

**Comment by wanda-carlson** [comment]
Sure, go ahead

File: src/bert.py [hunk 1]
```diff
@@ -26,3 +26,4 @@
 # 2 0.8.8 Preflight Core
 # 3 0.8.8 Preflight Core
  - multiple assigned
+# 4 0.8.8 Preflight Core
```
Comment (wanda-carlson) [inline src/bert.py:29]: Can we rename this?

File: src/bert.py [hunk 2]
```diff
@@ -26,3 +26,4 @@
 # 2 0.8.8 Preflight Core
 # 3 0.8.8 Preflight Core
  - multiple assigned
```
Comment (wanda-carlson) [inline src/bert.py:28]: Also add a commit with special characters?

**Review by wanda-carlson — CHANGES_REQUESTED** [review]
Minor changes

File: src/bert.py (hunk 1 above)
Comment (henry-richard) [reply src/bert.py:29]: I dont think we need to do that

**Review by wanda-carlson — APPROVED** [review]
lgtm

**Pull request merged**